src/localAnsibleRunner.ts

```diff
@@ -31,7 +31,21 @@ export class LocalAnsibleRunner extends TerminalBaseRunner {
         if (utilities.isTelemetryEnabled()) {
             cmdsToTerminal.push(envCmd + Constants.UserAgentName + '=' + utilities.getUserAgent());
         }
-        cmdsToTerminal.push(this.getRunPlaybookCmd("\"" + playbook + "\""));
+
+        if (utilities.isWslEnabled()) {
+            var sourcePath = path.dirname(playbook);
+            var targetPath = '/mnt/' + playbook.replace(/:/, '');
```

Comment:
`:` to `/`, not empty

Reply:
I have to disagree. Absolute paths in windows are <drive>:\path. You don't want to replace : with / or you would end up with /mnt/c//something.

Reply:
got it.

Reply:
> how do you do the testing?

I test on Windows 10 Pro 1903 compiling the extension from source and validating against WSL running Ubuntu 18.04 testing w/ using the WSL instance as both a task engine and local.